### PR TITLE
Messages

### LINKED ISSUE #3: Add home page (html)
Home page with list of messages and button send. Add tests

### PR DESCRIPTION
**Pull request by SerhiiPoliakov1**
Create entity Message , MessageController and home page with list of messages and button send.

### CODE REVIEW

File: src/main/java/co/norse/chat/controller/MessageController.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Controller
+public class MessageController {
+    List<Message> allMessage = new ArrayList<>();
```
Comment (bsolomko) [inline src/main/java/co/norse/chat/controller/MessageController.java]: rename to messages

File: src/main/java/co/norse/chat/model/Message.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
+package co.norse.chat.model;
+
+public class Message {
```
Comment (bsolomko) [inline src/main/java/co/norse/chat/model/Message.java:8]: nit: format

File: src/main/resources/templates/home.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        <label for="m">Message:</label >
+        <input type="text" th:field="*{message}" id="m">
+        <br/>
+        <input type="submit" value="Send">
+    </form>
+</div>
+
+
+</body>
+</html>
```
Comment (bsolomko) [inline src/main/resources/templates/home.html:40]: add empty line

File: src/main/java/co/norse/chat/model/Message.java (hunk 2 above)
Comment (bsolomko) [inline src/main/java/co/norse/chat/model/Message.java:8]: add hashCode equals toString

File: src/main/java/co/norse/chat/controller/MessageController.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Controller
+public class MessageController {
```
Comment (bsolomko) [inline src/main/java/co/norse/chat/controller/MessageController.java:16]: add tests

**Review by bsolomko — CHANGES_REQUESTED** [review]

File: src/main/java/co/norse/chat/controller/MessageController.java (hunk 1 above)
Comment (SerhiiPoliakov1) [reply src/main/java/co/norse/chat/controller/MessageController.java]: I did delete this field,its don`t exist

Comment (bsolomko) [reply src/main/java/co/norse/chat/controller/MessageController.java]: I have deleted this field, it doesn't exist

**Review by bsolomko — APPROVED** [review]

**Pull request merged**